### DISCUSSION TITLE
Interior temperature - sometimes wrong?

### DISCUSSION THREAD

**Discussion by E46Fanatic** [BimmerGestalt/AAIdrive#599] [Q&A]
Hi there. I have a question regarding the interior temp. figure in the app (car information). Sometimes it reads wrong values. Sometimes its a bit higher (like 5 °C, so maybe its really a bit warmer near the sensor, dont know), but sometimes it drops to about 10 °C (and stuck at that until car restart). Sometimes its correct.
Is it possible that the info in the app is wrong? Or my interior temp. sensor is failing? While in Italy (very hot), my outside (ambient) temp.  sensor was failing (stuck at 50 °C, check engine on sometimes, diagnostics said that there is a short in ambient temp. sensor). Can it be connected with that (maybe the intrior temp. is being calculated somehow using the info from the ambient temp. sensor)? Although it has to be noted that the ambient temp. sensor started working normally again after a few days and has been ok for about a week now. 
Thanks!

**Comment by hufman** [comment]
That's interesting... AAIdrive merely reports whatever the car api says, at which point it's up to the car's internals!
